packages/astro/src/core/app/index.ts

```diff
@@ -54,6 +54,20 @@ export interface RenderOptions {
 	 */
 	locals?: object;
 
+	/**
+	 * A custom fetch function for retrieving prerendered pages, such as 404 or 500 error pages.
+	 *
+	 * Default: `fetch` 
```

Comment:
There's no default for this option, actually. The documentation should specify that if not provided, Astro won't use it and fallback to its default behaviour

Reply:
https://github.com/withastro/astro/pull/13403/files#diff-bfed2e2dbd4a6b01a9ef9d354a4b89031a3c070d5fa462e8232e59555ae3352aR305

The default comes from here, would you prefer to refactor it to something else? what would it be?

Reply:
It's more about expectations, not code.

Documenting that we use fetch isn't needed to the user, because we make them assume that the custom function must behave like `fetch`, which isn't necessarily true. A custom adapter might be able to do a call to the file system, and just return `Promise.resolve`. I hope it makes sense.

Reply:
I got you now, thank you for clarifying, I'll change the phrasing.